src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -290,6 +293,23 @@
   @end
 @end
 
+@private processDoc(doc)
+  # If there are no parameters, we generate the method description in one line.
+  @if doc.parameters.isEmpty
+    /** {@doc.description} */
```

Comment:
The formatter takes care of long lines, correct? (In all languages?)

Reply:
Not really. we'll have to handle long lines in gapic-generator. I'm gonna push some new changes to handle them.